server/chainquery/queries/claimQueries.js

```diff
@@ -0,0 +1,184 @@
+const logger = require('winston');
+
+const returnShortId = (claimsArray, longId) => {
```

Comment:
Is this the canonical url?

Reply:
If so this doesn't seem right. You want the claim_name included here. Also the claim_id is a hash not based on the height. Ignore if this is not the case. @kauffj has a preference for how we do these shortened URLs.

Reply:
This is the existing logic, I believe. I think there are some issues with it, but we're going to handle it through https://github.com/lbryio/spee.ch/issues/184